build.gradle

```diff
@@ -43,6 +43,7 @@ apply plugin: "me.champeau.gradle.jmh"
 apply plugin: "com.github.hierynomus.license"
 apply plugin: "com.jfrog.bintray"
 apply plugin: "com.jfrog.artifactory"
+apply plugin: "eclipse"
```

Comment:
why do we need the eclipse plugin?

Reply:
This is so I can run `./gradlew eclipse` for import into Eclipse without using gradle plugin. It's not required for the general build obviously but is a convenience for me. Can remove if has any negative effects.

Reply:
Works for me.